### ISSUE TITLE
Setting `Chart.has_title = False` doesn't work with doughnut

### ISSUE THREAD

**Issue by jmm** [scanny/python-pptx#408]
Hello! Thanks for the work on this project 🙌

This is with python-pptx@0.6.10. [Repro](https://github.com/jmm/python-pptx-repro-doughnut-chart-title)

```python
chart_data = ChartData()
chart_data.categories = ['Yes', 'No']
chart_data.add_series('Series 1', (42, 24))

for chart_type in [XL_CHART_TYPE.PIE, XL_CHART_TYPE.DOUGHNUT]:
    for has_title in [True, False]:
        slide = presentation.slides.add_slide(presentation.slide_layouts[1])
        placeholder = slide.placeholders[13]
        chart = placeholder.insert_chart(chart_type, chart_data).chart
        chart.has_title = has_title
```

**Comment by scanny** [comment]
Ah, yes, this is a known glitch. It happens on some other types as well. For some (undocumented) reason, on some chart types, PowerPoint uses the first series name as the title for a chart if it doesn't have the optional `<c:title>` element.

Try:

```python
chart.chart_title.has_text_frame = True
```

and see if that clears it up; it does on mine. Not sure why exactly, but if you want to dig into the XML it would be nice to know what condition triggers this so we could possibly fix it.

**Comment by scanny** [comment]
Okay, so here's the logic that I think makes sense of this behavior. Just to recap, in a chart (of at least certain types) that has only _one_ series, the name of the first series appears as the title of the chart without you actually asking PowerPoint to do that.

Leaving aside the question of whether this behavior is _justified_, the reason why this might be _reasonable_ is that the _title can substitute for the legend in this single-series case_.

Generally, a legend maps a colors or symbol to a series, so you can tell which series is which. In the single-series case, you still need the name (what the values represent) but you no longer need the color-patch because there's only one, or because the series uses a different color for each value (bar, segment) (common with pie and I expect doughnut charts, but also an option with bar charts).

So a useful option in this single-series case is to turn the legend off and let the title explain what the chart is representing. Anyway, opinions could differ on whether that's "helping", but at least it makes sense to me now why they might have done this on purpose.

**Comment by scanny** [comment]
Should be fixed in release v0.6.13 later today.

**Comment by jmm** [comment]
Thanks for fixing that, for the detailed breakdown, and the workaround! (And sorry for the delay saying that.) That is interesting about chart title substituting for legend for single-series charts.
